Automattic/simplenote-ios
# Update Search Highlight Colors

This PR updates search matches highlight colors in the note editor.

P.S. In addition I updated `NSString (Search)` category to filter array using a closure.

@jleandroperez Are you up for another PR? :-) Thanks!

Closes #954 

### Test
1. Open note list
2. Search for something (that's in one of the notes)
3. Open one of the notes

- [ ] Check that highlight colors matches with the mockups in #954
